src/main.c

```diff
@@ -49,13 +51,13 @@ int main() {
 #endif
     if(PWR_CheckPowerSwitch()) PWR_Shutdown();
 
-    LCD_Clear(0x0000);
-#ifdef TEST_ADC
-    ADC_ScanChannels(); while(1);
-#endif
-    u32 buttons = ScanButtons();
-    if(CHAN_ButtonIsPressed(buttons, BUT_ENTER) || !FS_Mount(NULL, NULL)) {
-        LCD_DrawUSBLogo(LCD_WIDTH, LCD_HEIGHT);
+    LCD_Clear(0x0000); //diff8
+                       //diff9
+                       //diff10
+                       //diff11
+    u32 buttons = ScanButtons(); //diff12
+    if(CHAN_ButtonIsPressed(buttons, BUT_ENTER) || !FS_Mount(NULL, NULL)) { //diff13
```

Comment:
Lint Error:
main.c:54:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:54:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:55:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:56:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:57:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:58:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:58:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:59:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:59:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:59:  Missing space before ( in if(  [whitespace/parens] [5]